exercises/04-integration/after/Divergent.Customers.API/Divergent.Customers.API.csproj

```diff
@@ -211,7 +210,7 @@
         <WebProjectProperties>
           <UseIIS>True</UseIIS>
           <AutoAssignPort>True</AutoAssignPort>
-          <DevelopmentServerPort>20185</DevelopmentServerPort>
+          <DevelopmentServerPort>20186</DevelopmentServerPort>
```

Comment:
Why is this change required? Is it related to the package upgrades?

Reply:
No, not related to the package update, but it didn't run properly without this change...

Reply:
I'll do the before as well, and squash all to one commit once all is good

Reply:
Please leave this change as a separate commit.

Reply:
BTW - I think we need a code change as a result of this. IIRC the host and port of each web API is hardcoded in C#. @Particular/microservices-workshop-maintainers am I correct?

What I can't understand is why I have had no trouble running the solution. What problem did you encounter before changing these ports?

Reply:
I had port collisions... so I looked at all the api endpoints and made sure they are correct...

Reply:
I don't have this problem. Everything runs OK when I run the solution as-is. I don't think I saw any workshop attendees having this problem either. @Particular/microservices-workshop-maintainers does anyone know why @seanfarmar is having this problem?

Regardless, would it be best to remove it from this PR and deal with this separately? If we can keep this PR isolated to package updates only then we can get it merged in quicker.